packages/block-library/src/media-text/edit.js

```diff
@@ -226,7 +226,7 @@ function MediaTextEdit( { attributes, isSelected, setAttributes, clientId } ) {
 	};
 
 	const imageSizeOptions = map(
-		filter( imageSizes, ( { slug } ) =>
+		imageSizes.filter( ( { slug } ) =>
```

Comment:
Defaults to an array:

https://github.com/WordPress/gutenberg/blob/a4040e40965b5430874413e5d5389ac889ad13b9/packages/block-editor/src/store/defaults.js#L133